pkg/webui/locales/en.json

```
@@ -105,7 +114,12 @@
   "account.views.login.index.loginToContinue": "Please login to continue",
   "account.views.login.index.loginFailed": "Login failed",
   "account.views.login.index.accountDeleted": "Account deleted",
+  "account.views.oauth-authorization-settings.index.deleteButton": "De-authorize this client",
+  "account.views.oauth-authorization-settings.index.deleteSuccess": "This oauth client was successfully de-authorized",
+  "account.views.oauth-authorization-settings.index.deleteFailure": "There was an error and this client could not be de-authorized",
   "account.views.oauth-client-add.index.addClient": "Add OAuth Client",
+  "account.views.oauth-client-authorization-overview.index.authorizationSettings": "Authorization settings",
+  "account.views.oauth-client-authorization-overview.index.authorizationTokens": "Active authorization tokens",
   "account.views.overview.index.accountAppInfoTitle": "Welcome, {userId}! 👋",
   "account.views.overview.index.accountAppInfoMessage": "<p>You have successfully logged into the Account App. The Account App is the official user account management application of The Things Stack. In the near future, you will additionally be able to use this application to</p>\n<ul><li>Manage your active sessions</li><li>Manage your OAuth authorizations</li></ul>",
```

I realize that we should update this message as well, since the functions it mentions are now available.
```suggestion
  "account.views.overview.index.accountAppInfoMessage": "<p>You have successfully logged into the Account App. The Account App is the official user account management application of The Things Stack.</p>",
```